holoviews/util/settings.py

```diff
@@ -141,40 +141,46 @@ class OutputSettings(KeywordSettings):
     """
 
     # Lists: strict options, Set: suggested options, Tuple: numeric bounds.
-    allowed = {'backend'     : list_backends(),
-               'fig'         : list_formats('fig'),
-               'holomap'     : list_formats('holomap'),
-               'widgets'     : ['embed', 'live'],
-               'fps'         : (0, float('inf')),
-               'max_frames'  : (0, float('inf')),
-               'max_branches': {None},            # Deprecated
-               'size'        : (0, float('inf')),
-               'dpi'         : (1, float('inf')),
-               'filename'    : {None},
-               'info'        : [True, False],
-               'css'         : {k: basestring
-                                for k in ['width', 'height', 'padding', 'margin',
-                                          'max-width', 'min-width', 'max-height',
-                                          'min-height', 'outline', 'float']}}
-
-    defaults = OrderedDict([('backend'     , None),
-                            ('fig'         , None),
-                            ('holomap'     , None),
-                            ('widgets'     , None),
-                            ('fps'         , None),
-                            ('max_frames'  , 500),
-                            ('size'        , None),
-                            ('dpi'         , None),
-                            ('filename'    , None),
-                            ('info'        , False),
-                            ('css'         , None)])
+    allowed = {'backend'       : list_backends(),
```

Comment:
Ok, I think this addresses my worry about the output magic working - looks like it was updated here after all.